packages/live-share/src/EphemeralObjectSynchronizer.ts

```diff
@@ -86,11 +87,11 @@ export class EphemeralObjectSynchronizer<TState extends object> {
      * @param getState A function called to retrieve the objects current state. This will be called prior to a "connect" or "update" message being sent.
      * @param updateState A function called to process a state update received from a remote instance. This will be called anytime a "connect" or "update" message is received.
      */
-    constructor(id: string, containerRuntime: IContainerRuntimeSignaler, getState: GetSynchronizationState<TState>, updateState: UpdateSynchronizationState<TState>) {
+    constructor(id: string, runtime: IRuntimeSignaler, containerRuntime: IContainerRuntimeSignaler, getState: GetSynchronizationState<TState>, updateState: UpdateSynchronizationState<TState>) {
```

Comment:
nit: did you add an `@param` comment for the new `runtime` parameter? Could get confusing now that there are two different runtimes.

Reply:
good catch. just an oversight.